.github/workflows/wait_for_required_workflows.yml

```diff
@@ -52,7 +52,7 @@ jobs:
               const checkRuns = await github.paginate(github.rest.checks.listForRef, {
                 owner: 'cloudquery',
                 repo: 'cloudquery',
-                ref: context.sha,
```

Comment:
This is the wrong sha for pull requests events:
![image](https://user-images.githubusercontent.com/26760571/193079425-3a4aeb46-346e-4b80-8aab-7fa347f07873.png)

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request